### PR TITLE
Update LiveSplit.AutoSplitters.xml

### PR DESCRIPTION
**Pull request by Syn34**
If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [ x] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [x ] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [x ] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

**Comment by just-ero** [comment]
I ask that you please do not include the splitter's version in the description here. It will be annoying for both you and us if you have to PR this repo on every little change you make to the auto splitter.

**Comment by Syn34** [comment]
Sorry about that I never considered it. Should I take out the versions and request the pull again so we don't run into this again? Thanks

**Comment by just-ero** [comment]
You don't have to open another pull request, simply edit this one. Remember to change the file name on your splitter's repo as well.

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -5,10 +5,10 @@
             <Game>Ratchet &amp; Clank: Rift Apart</Game>
         </Games>
         <URLs>
-            <URL>https://raw.githubusercontent.com/Syn34/Rift-Apart-Autosplitter/main/Riftsplitter%20v1.728.0.0%20by%20Jasper%20and%20Sin.asl</URL>
+            <URL>https://raw.githubusercontent.com/Syn34/Rift-Apart-Autosplitter/main/Riftsplitter.asl</URL>
         </URLs>
         <Type>Script</Type>
-        <Description>Riftsplitter v1.728.0.0 by Jasper and Sin (Splitting/Load Removal)</Description> 
+        <Description>Riftsplitter by Jasper & Sin (Splitting/Load Removal)</Description>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml]:
```suggestion
        <Description>Riftsplitter by Jasper &amp; Sin (Splitting/Load Removal)</Description> 
```

**Pull request merged**